src/sklearn_evaluation/tracker.py

```diff
@@ -378,8 +379,31 @@ def upsert(self, uuid, parameters):
         cur.close()
         self.conn.commit()
 
+    @SKLearnEvaluationLogger.log("SQLiteTracker")
     def upsert_append(self, uuid, parameters):
-        """Append the parameters to an existing experiment"""
+        """Append the parameters to an existing experiment
+
+        Examples
+        --------
+        >>> from sklearn_evaluation import SQLiteTracker
+        >>> tracker = SQLiteTracker('experiments.db')
+        >>> exp = tracker.new_experiment()
+
+        >>> #Log initial metric_a values for the experiment
+        >>> exp.log("metric_a", [0.8, 0.85]) # doctest: +SKIP
+        metric_a: [0.8, 0.85]
+
+        >>> #Append new "metric_a" values and adding "metric_b" values
+        >>> tracker.upsert_append(
+        ...        exp.uuid,
+        ...        dict(
+        ...            metric_a=[0.9, 0.95],
```

Comment:
Please replace this with this metric_a = 0.9 (instead of the list)

Reply:
Done!